File: samples/client/petstore/cpp-tiny/lib/Models/ExampleResponse.cpp


#include "ExampleResponse.h"

using namespace Tiny;

ExampleResponse::ExampleResponse()
{
	emptyString = std::string();
	numberString = std::string();
	boolString = std::string();
	nullString = std::string();
	aBool = bool(false);
	zero = int(0);
}

ExampleResponse::ExampleResponse(std::string jsonString)
{
	this->fromJson(jsonString);
}

ExampleResponse::~ExampleResponse()
{

}

void
ExampleResponse::fromJson(std::string jsonObj)
{
    bourne::json object = bourne::json::parse(jsonObj);

    const char *emptyStringKey = "emptyString";

    if(object.has_key(emptyStringKey))
    {
        bourne::json value = object[emptyStringKey];



        jsonToValue(&emptyString, value, "std::string");


    }

    const char *numberStringKey = "numberString";

    if(object.has_key(numberStringKey))
    {
        bourne::json value = object[numberStringKey];



        jsonToValue(&numberString, value, "std::string");


    }

    const char *boolStringKey = "boolString";

    if(object.has_key(boolStringKey))
    {
        bourne::json value = object[boolStringKey];



        jsonToValue(&boolString, value, "std::string");


    }

    const char *nullStringKey = "nullString";

    if(object.has_key(nullStringKey))
    {
        bourne::json value = object[nullStringKey];



        jsonToValue(&nullString, value, "std::string");


    }

    const char *aBoolKey = "aBool";

    if(object.has_key(aBoolKey))
    {
        bourne::json value = object[aBoolKey];



        jsonToValue(&aBool, value, "bool");


    }

    const char *zeroKey = "zero";

    if(object.has_key(zeroKey))
    {
        bourne::json value = object[zeroKey];



        jsonToValue(&zero, value, "int");


    }


}

bourne::json
ExampleResponse::toJson()
{
    bourne::json object = bourne::json::object();





    object["emptyString"] = getEmptyString();






    object["numberString"] = getNumberString();






    object["boolString"] = getBoolString();






    object["nullString"] = getNullString();






    object["aBool"] = isABool();






    object["zero"] = getZero();



    return object;

}

std::string
ExampleResponse::getEmptyString()
{
	return emptyString;
}

void
ExampleResponse::setEmptyString(std::string  emptyString)
{
	this->emptyString = emptyString;
}

std::string
ExampleResponse::getNumberString()
{
	return numberString;
}

void
ExampleResponse::setNumberString(std::string  numberString)
{
	this->numberString = numberString;
}

std::string
ExampleResponse::getBoolString()
{
	return boolString;
}

void
ExampleResponse::setBoolString(std::string  boolString)
{
	this->boolString = boolString;
}

std::string
ExampleResponse::getNullString()
{
	return nullString;
}

void
ExampleResponse::setNullString(std::string  nullString)
{
	this->nullString = nullString;
}

bool
ExampleResponse::isABool()
{
	return aBool;
}

void
ExampleResponse::setABool(bool  aBool)
{
	this->aBool = aBool;
}

int
ExampleResponse::getZero()
{
	return zero;
}

void
ExampleResponse::setZero(int  zero)
{
	this->zero = zero;
}

File: samples/server/petstore/java-helidon-server/se/src/main/java/org/openapitools/server/model/ExampleResponse.java
package org.openapitools.server.model;



/**
 * dummy
 */
public class ExampleResponse   {

    private String emptyString = "";
    private String numberString = "42";
    private String boolString = "false";
    private String nullString = "null";
    private Boolean aBool = false;
    private Integer zero = 0;

    /**
     * Default constructor.
     */
    public ExampleResponse() {
    // JSON-B / Jackson
    }

    /**
     * Create ExampleResponse.
     *
     * @param emptyString emptyString
     * @param numberString numberString
     * @param boolString boolString
     * @param nullString nullString
     * @param aBool aBool
     * @param zero zero
     */
    public ExampleResponse(
        String emptyString, 
        String numberString, 
        String boolString, 
        String nullString, 
        Boolean aBool, 
        Integer zero
    ) {
        this.emptyString = emptyString;
        this.numberString = numberString;
        this.boolString = boolString;
        this.nullString = nullString;
        this.aBool = aBool;
        this.zero = zero;
    }



    /**
     * Get emptyString
     * @return emptyString
     */
    public String getEmptyString() {
        return emptyString;
    }

    public void setEmptyString(String emptyString) {
        this.emptyString = emptyString;
    }

    /**
     * Get numberString
     * @return numberString
     */
    public String getNumberString() {
        return numberString;
    }

    public void setNumberString(String numberString) {
        this.numberString = numberString;
    }

    /**
     * Get boolString
     * @return boolString
     */
    public String getBoolString() {
        return boolString;
    }

    public void setBoolString(String boolString) {
        this.boolString = boolString;
    }

    /**
     * Get nullString
     * @return nullString
     */
    public String getNullString() {
        return nullString;
    }

    public void setNullString(String nullString) {
        this.nullString = nullString;
    }

    /**
     * Get aBool
     * @return aBool
     */
    public Boolean getaBool() {
        return aBool;
    }

    public void setaBool(Boolean aBool) {
        this.aBool = aBool;
    }

    /**
     * Get zero
     * @return zero
     */
    public Integer getZero() {
        return zero;
    }

    public void setZero(Integer zero) {
        this.zero = zero;
    }

    /**
      * Create a string representation of this pojo.
    **/
    @Override
    public String toString() {
        StringBuilder sb = new StringBuilder();
        sb.append("class ExampleResponse {\n");
        
        sb.append("    emptyString: ").append(toIndentedString(emptyString)).append("\n");
        sb.append("    numberString: ").append(toIndentedString(numberString)).append("\n");
        sb.append("    boolString: ").append(toIndentedString(boolString)).append("\n");
        sb.append("    nullString: ").append(toIndentedString(nullString)).append("\n");
        sb.append("    aBool: ").append(toIndentedString(aBool)).append("\n");
        sb.append("    zero: ").append(toIndentedString(zero)).append("\n");
        sb.append("}");
        return sb.toString();
    }

    /**
     * Convert the given object to string with each line indented by 4 spaces
     * (except the first line).
    */
    private static String toIndentedString(Object o) {
        if (o == null) {
          return "null";
        }
        return o.toString().replace("\n", "\n    ");
    }
}

